_rules/html-page-lang-xml-lang-match-5b7ae0.md

```diff
@@ -19,19 +19,22 @@ authors:
 
 ## Applicability
 
-The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element with both `lang` and `xml:lang` attributes that have [valid language subtags](#valid-language-subtag).
+This rule applies to any [document element](https://dom.spec.whatwg.org/#document-element) if it is an `html` element that:
 
-**Note**: Documents embedded into other documents, such as through `iframe` or `object` elements are not applicable because they are not web pages according to the definition in WCAG.
+- is in a [top-level browsing context](https://html.spec.whatwg.org/#top-level-browsing-context); and
+- has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html`; and
+- has a `lang` attribute that has a [valid language subtag](#valid-language-subtag); and
+- has an `xml:lang` attribute.
 
-## Expectation
+**Node:** `html` elements within `iframe` and `object` elements are not applicable as `iframe` and `object` elements create [nested browsing contexts](https://html.spec.whatwg.org/#nested-browsing-context).
 
-For each test target, the value of the primary language subtag (characters before the first dash) for the `lang` and `xml:lang` attributes are the same.
+## Expectation
 
-**Note**: HTML 5 specification requires the `lang` and `xml:lang` attributes to match exactly. This is not known to impact accessibility, which is why it is permitted in this rule.
+For each test target, the values of the [primary language subtags](https://tools.ietf.org/html/bcp47#section-2.2.1) and the [extended language subtags](https://tools.ietf.org/html/bcp47#section-2.2.2), if any exist, for the `lang` and `xml:lang` attributes are the same.
```

Comment:
I don't think this PR should add the extended language subtags here. I'm still not convinced that should be required at all, but even if it is, that should be in a different PR. Here's why i disagree this is required:

An element with `lang="nl-NL" xml:lang="nl-BE"` does not create accessibility problems. While I agree with you that there are scenarios in which having a mismatch between the extended subtags can be a problem, it definitely isn't always a problem. So we shouldn't always fail it. Passing the rule does not mean the SC is satisfied. So even in scenarios where the primary subtag matches and the extended subtags don't, and this does cause an accessibility issue, this rule is still correct.